src/model/groups.rs

```diff
@@ -114,3 +116,78 @@ pub struct GroupAuditLogDataChange<T> {
 	/// The new field after the change.
 	pub new: T,
 }
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+/// Details about a group un-ban.
+pub struct GroupBan {
+	/// Unique identifier of the un-ban.
+	pub id: String,
+	/// Identifier of the group.
+	pub group_id: crate::id::Group,
+	/// Identifier of the user who was un-banned.
+	pub user_id: crate::id::User,
+	/// Flag indicating if the user was representing the group at the time of
+	/// un-ban.
+	pub is_representing: bool,
+	/// List of role identifiers the user had in the group.
+	pub role_ids: Vec<Value>,
+	/// List of managed role identifiers the user had in the group.
+	pub m_role_ids: Vec<Value>,
+	/// Time of when the user joined.
+	pub joined_at: Option<String>,
```

Comment:
I guess this is fine, though imagine that this probably should be an optional `OffsetDateTime`...?